.github/workflows/ci.yaml

```diff
@@ -0,0 +1,43 @@
+name: CI Workflow
+
+on: [push]
+
+jobs:
+
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:9.6
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: audit_test
+        ports:
+        - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
```

Comment:
Do we want `setup-python@v1`? Does that provide `3.7` by default? I'd think we would want to target `3.9.x` and it seems like the action is at `v2` now https://github.com/actions/setup-python

Reply:
`with: python-version: 3.7` sets up 3.7

it looks like v2 just adds support for things we don't need (https://github.com/actions/setup-python#whats-new)

as to using a more recent version of python, we could definitely do it here and in other services, but since it's not high priority i'd rather not do that in this PR

Reply:
Gotcha ✔️ 

And thanks! I didn't see the `with:` directive just below 👀